Test/scripts/ci-magento-integration.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+set -x
+
+trap '>&2 echo Error: Command \`$BASH_COMMAND\` on line $LINENO failed with exit code $?' ERR
+
+Test/scripts/install_magento.sh
+
+cd ..
+mkdir -p magento/app/code/Bolt/Boltpay
+# magento requires the code to be in the magento installation dir
+# However if we copy codecov gets confused because of multiple sources.
+# So a quick fix is to keep a copy of the composer.json
+# TODO(roopakv): Initialize circle with the repo in the right place
+mv project/* magento/app/code/Bolt/Boltpay/
+mkdir -p project
+cp magento/app/code/Bolt/Boltpay/composer.json project/composer.json
+cp magento/app/code/Bolt/Boltpay/Test/Unit/integration_phpunit.xml magento/dev/tests/integration/bolt_phpunit.xml
+
+
+echo "Creating DB for integration tests"
+mysql -uroot -h 127.0.0.1 -e 'CREATE DATABASE magento_integration_tests;'
+
+
+cd magento/dev/tests/integration/
+cp etc/install-config-mysql.php.dist etc/install-config-mysql.php
+sed -i 's/localhost/127.0.0.1/g' etc/install-config-mysql.php
+sed -i 's/123123q//g' etc/install-config-mysql.php
+sed -i '/amqp/d' etc/install-config-mysql.php
```

Comment:
What's happening with these two sed commands? How does the first know where the replace? and what's the second doing?

Reply:
So **123123q** is db password from example config. We use an empty password in circle-ci.
The second removes all lines related to amqp -  Advanced Message Queuing Protocol. We don't use it in test flow

File:
https://github.com/magento/magento2/blob/2.4-develop/dev/tests/integration/etc/install-config-mysql.php.dist